zipline/gens/brokers/ib_broker.py

```diff
@@ -456,9 +459,11 @@ def fundamentalData(self, req_id, data):
     def marketDataType(self, req_id, market_data_type):
         log_message('marketDataType', vars())
 
-    def realtimeBar(self, req_id, time, open_, high, low, close, volume, wap,
+    def realtimeBar(self, req_id, time, open, high, low, close, volume, wap,
                     count):
-        log_message('realtimeBar', vars())
+        value = (";".join([str(close), str(count), str(time), str(volume),
```

Comment:
the str(volume) will not work here, as tick_type=48 returns the total_volume.
We'd need to do bookkeeping to be in sync with that behavior.